### ISSUE TITLE
Multi-line watermark

### ISSUE THREAD

**Issue by art2crazy** [hhrutter/pdfcpu#61]
Hi Horst,
Great project! I just started to learn go, so if I missed anything, sorry.
I'm using api and pdfcpu pkgs to add a watermark to the pdf, like so:
```
str := "Line1\nLine2"
wm, err := pdfcpu.ParseWatermarkDetails(str, true)
// check err
in := "somefile.pdf"
out := "watermark.pdf"
cmd := api.Command{InFile: &in, OutFile: &out, Watermark: wm}
_, err = api.AddWatermarks(&cmd)
// check err
```
Everything works fine, except the new line in the string is ignored. 
So, the question: is this possible to have a multi-line watermark, and if so, what am I missing?

art

**Comment by hhrutter** [comment]
Hi there!

You did not miss anything.
Multiline watermarks are just not supported yet.

**Comment by hhrutter** [comment]
Fixed with [v0.1.23](https://github.com/hhrutter/pdfcpu/releases/tag/v0.1.23)
